### PR TITLE
Use ElementModals for a few front modals

### PR DESCRIPTION
**Pull request by philipperolet**
## Description

Following PR #3708 , use the new ElementModal component

## Risk
Low. Tested.

### CODE REVIEW

File: front/package-lock.json [hunk 1]
```diff
@@ -4,8 +4,9 @@
   "requires": true,
   "packages": {
     "": {
+      "name": "front",
       "dependencies": {
-        "@dust-tt/sparkle": "^0.2.92",
+        "@dust-tt/sparkle": "^0.2.92-pr",
```
Comment (flvndvd) [inline front/package-lock.json:9]: Weird.

File: front/pages/w/[wId]/assistant/[cId]/index.tsx [hunk 2]
```diff
@@ -110,10 +109,9 @@ export default function AssistantConversation({
     const handleRouteChange = () => {
       const assistantSId = router.query.assistantDetails ?? [];
       if (assistantSId && typeof assistantSId === "string") {
-        setIsAssistantDetailOpen(true);
         setDetailViewContent(assistantSId);
       } else {
-        setIsAssistantDetailOpen(false);
+        setDetailViewContent("");
```
Comment (flvndvd) [inline front/pages/w/[wId]/assistant/[cId]/index.tsx:114]: Shouldn't it be `null`?

**Review by flvndvd — COMMENTED** [review]
Can you rebase?

File: front/pages/w/[wId]/assistant/[cId]/index.tsx (hunk 2 above)
Comment (philipperolet) [reply front/pages/w/[wId]/assistant/[cId]/index.tsx:114]: no setDetailViewContent is a string for the assitant Id (historically doesn't take null, not sure why)

File: front/package-lock.json (hunk 1 above)
Comment (philipperolet) [reply front/package-lock.json]: fixed

File: front/pages/w/[wId]/assistant/[cId]/index.tsx (hunk 2 above)
Comment (philipperolet) [reply front/pages/w/[wId]/assistant/[cId]/index.tsx:114]: do you think we should change it? string | null?

Comment (flvndvd) [reply front/pages/w/[wId]/assistant/[cId]/index.tsx:114]: "" is falsy 🤦

File: front/pages/w/[wId]/builder/assistants/index.tsx [hunk 3]
```diff
@@ -186,16 +186,12 @@ export default function WorkspaceAssistants({
...
-          assistantId={showDetails.sId}
-          show={showDetails !== null}
-          onClose={() => setShowDetails(null)}
-          mutateAgentConfigurations={mutateAgentConfigurations}
-        />
-      )}
-
+      <AssistantDetails
+        owner={owner}
+        assistantId={showDetails?.sId || null}
```
[the remark this answers is not in the document]
Comment (philipperolet) [reply front/pages/w/[wId]/builder/assistants/index.tsx:191]: image

**Pull request merged**